cli/root.go

```diff
@@ -130,6 +130,7 @@ func (r *RootCmd) CoreSubcommands() []*serpent.Command {
 		r.connectCmd(),
 		r.expCmd(),
 		gitssh(),
+		r.jail(),
```

Comment:
I haven't actually hidden the command yet but I didn't know if we might want it hidden at first.